components/script/dom/webglrenderingcontext.rs

```diff
@@ -236,10 +236,7 @@ impl WebGLRenderingContext {
             let max_vertex_attribs = ctx_data.limits.max_vertex_attribs as usize;
             Self {
                 reflector_: Reflector::new(),
-                webgl_sender: WebGLMessageSender::new(
-                    ctx_data.sender,
-                    window.get_event_loop_waker(),
-                ),
+                webgl_sender: WebGLMessageSender::new(ctx_data.sender, None),
```

Comment:
Sorry for this constant back and forth, but I think we need to chase this None all the way. Can you replace the `webgl_sender: WebGLMessageSender` with a simple `webgl_sender: WebGLChan` and remove all of the code around waking up the event loop, since it seems to be completely unused?

r=me with that change. Sorry again!

Reply:
oh no worries. I am aware. Actually I was writing, why I left it as None. See my comment here: https://github.com/servo/servo/pull/30101#discussion_r1294723047

Please let me know if you would prefer I do that changes in this same PR?